tesseract_ros/tesseract_examples/tesseract_ros_examples/src/car_seat_demo.cpp

```diff
@@ -27,14 +27,14 @@
 TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
 #include <jsoncpp/json/json.h>
 #include <ros/ros.h>
+#include <tesseract_msgs/ModifyEnvironment.h>
```

Comment:
For what it's worth, the car seat demo seems to crash on the return plan. Perhaps it is just finishing and dieing. I have not looked into it very much.